### ISSUE TITLE
Video rotation support 

### ISSUE THREAD

**Issue by taboca** [apc-io/apc-firefox-os#30]
I would like to use the Firefox OS infrastructure in vertical mode. I think it is interesting for users and it's a great use case.

**Comment by lechau-apc** [comment]
Yes, this might be a useful use case when the system runs on smartphones or tablets for example, but I dont think it's of the highest priority for Desktop based users. We might consider this at a later time, unless you can cite some concrete use cases to convince us, @taboca ?:)

**Comment by taboca** [comment]
Hi there - sorry for the late reply. The use case is this: http://www.telasocial.com/ my company and screens in spaces for events. Note: my first APC arrived, which one I won in Toronto (Mozilla event) so since I plan to start hacking in certain layers of the project — i can at least get involved a bit or contribute to tests if not code. Is this a very dependent part on Mozilla folks on you think it can be done on APC side? 

Business wise I am fine, so far, since the workaround is to rotate in HTML — it works for non interactive (no touch, no mouse) use cases.

**Comment by trungnt** [comment]
hi @taboca, so far, the APC 8950 board does not have any orientation sensor, so we can't have auto rotation feature. We haven't thought about the case that the board is used in portrait mode, so, we created a patch to fix orientation to landscape mode in any case. If, for now we support portrait mode, we may need to create a new patch. But however, we'll check if we can do this with a small change, that not affect to much to our patch.
